src/EnergyPlus/HeatRecovery.cc

```diff
@@ -101,45 +101,10 @@ namespace HeatRecovery {
     // U.S. Environmental Protection Agency software "SAVES" -
     //  School Advanced Ventilation Engineering Software http://www.epa.gov/iaq/schooldesign/saves.html
 
-    // OTHER NOTES: none
```

Comment:
Alright, this file is also vastly reworked, and looking at it on GitHub will probably not be worth it.  I'll summarize the changes here, and if you are interested, I suggest you look at it side-by-side locally with the develop branch version.
- Cleaned up lots of comment preambles in functions
- Eliminated nearly all `using` usages
- Implemented enum/switch for inputs where necessary
- Localized variable scope
- Used boolean assignment and/or ternary operators for variable assignment where simple IF/ELSE blocks were being 
- Converted all lookups into the vector of recovery units and performance data into references
- Converted free functions into member methods and changed accessing them as such, eliminating the index being passed around internally
- Changed `ExchTypeNum` to `ExchType` and removed struct member variables where possible
- Used BooleanSwitch with getEnumerationValue in GetInput for Yes/No input values
- Converted lists of scalar coefficients into std::array and cleaned up usage of them
- Added enum class for error code return from CalculateNTUfromEpsAndZ and handled it appropriately

That's about it, but that's a lot :smile: